legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/result/VoidRelationalResult.java

```diff
@@ -36,6 +36,11 @@ public class VoidRelationalResult extends Result
     private Statement statement;
 
     public VoidRelationalResult(MutableList<ExecutionActivity> activities, Connection connection, MutableList<CommonProfile> profiles)
+    {
+        this(activities, connection, profiles, true);
+    }
+
+    public VoidRelationalResult(MutableList<ExecutionActivity> activities, Connection connection, MutableList<CommonProfile> profiles, boolean logSQLWithParamValues)
```

Comment:
Is this constructor needed?

Reply:
Preserving since it's a public constructor